docs/release-notes/artifacts/pr0300.yaml

```
@@ -0,0 +1,20 @@
+# --- Release notes change artifact ----
+
+schema_version: 1
+changes:
+  - title: Add basic DDoS protection mechanisms with option to disable them.
```

Erin clarified the expected style for the title: https://canonical-platform-engineering.readthedocs-hosted.com/latest/engineering-practices/documentation/charm-release-notes/release-notes-templates/#style-considerations
```suggestion
  - title: Added basic DDoS protection mechanisms with option to disable them.
```